string name
string mac
string mode
WifiStationInfo station
